makefile

```diff
@@ -0,0 +1,5 @@
+install-dep:
+	pip install -r requirements.txt
+run:install-dep
+	flask -A student.py run --host=0.0.0.0 --port=4000
```

Comment:
Port should be configurable.
Apply 12 factor app methods to move it into Environment config.